### ISSUE TITLE
(my) own avatar overflows in new image view metadata

### ISSUE THREAD

**Issue by HarHarLinks** [element-hq/element-web#27185]
### Steps to reproduce

1. send image
2. open it

#### What happened instead?
in the metadata section describing the sender, my own avatar is offset to the bottom-right

### Application version

Element version: 54ba0873dada-react-c42562ef3967-js-8e0ef5ff2cd9 Crypto version: Rust SDK 0.7.0 (b1918e9), Vodozemac 0.5.1

### Will you send logs?

No

**Comment by t3chguy** [comment]
Duplicate of https://github.com/element-hq/element-web/issues/27175

**Comment by HarHarLinks** [comment]
@t3chguy 

> Duplicate of #27175

No, I'm aware of that, and my issue does not meet the description
> All UI elements should be within view

(they overflow the window) but only the avatar itself not being centered and larger than its circle cutout. i can spend some time crafting a screenshot without an actual face if you really need it.
